platform/engineering/design-docs/2021-01-13-web-component-library.md

```diff
@@ -0,0 +1,221 @@
+# Web Component library
+
+**Author(s):** Brooks Johnson  
+**Last Updated:** January 13, 2021  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** _Chris Valarida_ [x], _Dror Matalon_ [ ], _Michael Fleet_ [ ]
+
+
+## Overview
+
+### Objective
+
+The goal is to establish a component library using the [Web Components standard](https://developer.mozilla.org/en-US/docs/Web/Web_Components). This will eventually supercede the current `component-library` code and some of the liquid HTML templates currently in `vets-website`. A non-goal is creating an entirely new visual experience as part of the component library.
+
+The intended audience for this document is frontend engineers.
+
+### Background
+
+Currently, the design system has various implementations of "components" scattered in various places to serve various tools/libraries:
+
+- The `formation` npm package
+    - This contains static HTML, CSS, and vanilla JavaScript
+    - Used by copy-pasting markup onto static pages
+-  The `component-library` package
+    - This is the rebranded `formation-react`, containing React implementations of some patterns defined in `formation`
+    - Can only be used inside React apps
+- [Liquid templates in `vets-website`](https://github.com/department-of-veterans-affairs/vets-website/tree/958d89f18db29a1f3e1f1cbe4e304299eb387de7/src/site)
+    - This is similar to the static markup found in `formation`
+    - These templates of components are only used in the content build for va.gov
+
+Maintaining multiple codebases and sources of documentation for design system components is a burden for all teams involved and leads to an inconsistent user experience and wasted engineering time.
+
+### High Level Design
+
+The goal is to establish a single codebase and documentation for the implementation of design system components. This will allow the components to be used in React apps, static HTML, or any other web framework.
```

Comment:
This sentence is probably what I'm asking for in the Objective paragraph. Consider moving it up?